doc/changelog.rst

```diff
@@ -24,6 +24,8 @@ Usability improvements/Changes
 - Removes :code:`NEB().contains_peak()` in favour of :code:`NEB().images.contains_peak`
 - Modifies the :code:`CImages` constructor to ensure it's constructed from an :code:`Images` instance
 - Removes :code:`NEB.get_species_saddle_point()` in favour of :code:`NEB.peak_species`
+- Defines named constructors (:code:`from_endpoints(...)`, :code:`from_list(...)`) for :code:`NEB`
```

Comment:
Missing the underscore in `from_end_points()`

Reply:
thanks 😄